pkg/deviceclaimingserver/gateways/gateways.go

```diff
@@ -76,7 +87,7 @@ type Claimer interface {
 	// Claim claims a gateway.
 	Claim(ctx context.Context, eui types.EUI64, ownerToken string, clusterAddress string) error
 	// Unclaim unclaims a gateway.
-	Unclaim(context.Context, types.EUI64, string) error
+	Unclaim(ctx context.Context, eui types.EUI64) error
```

Comment:
The third parameter was ambiguous: there was code that passed the cluster address and code that passed the owner token. I don't think we need either.